wg-lts/charter.md

```diff
@@ -0,0 +1,95 @@
+- Creating and prioritizing a list of areas that require investments to improve long term supportability. This could include:
+  * Upgrade path considerations.
+  * Costs of Kubernetes releases in terms of:
+    * Infrastructure
+    * People
+- Initiate and drive cross-SIG changes related to long term support.
+- Identify ways to that vendors and other community members can better support Kubernetes releases.
+
+### Out of scope
+
+- The lifecycle of projects outside of the Kubernetes org.
+- Designing and executing on changes clearly falling into individual SIG
+  responsibilities. This is a working group, no code implementation is the responsibility of this Working Group.
+- Technical and end-user support:  The WG may make recommendations
+  around support to those responsible for relevant code and responsible
+  for the release engineering operations and automation, but does not
+  own code itself.
+
+## Special Powers
+
+None
+
+## Stakeholders
+
+Any changes identified by this working group will require involvement and investment from the following SIGS:
+
+- SIG Architecture
+  High-level input on requirements.
+- SIG Cluster Lifecycle
+  Input on cluster upgrade mechanics.
+- SIG K8s Infra
+  Input on infrastructure costs related to increased support period(s).
+- SIG Release
+  Input on maintaining older branches and additional releases.
+- SIG Testing
+  Input on testing impacts related to upgrades and longer test periods. 
+  
+Improvements identified will likely involve every SIG, but the list above are identified as the primary stakeholders.
+
+## Deliverables
+
+The artifacts the group is supposed to deliver include:
+- Survey results better describing Kubernetes version use patterns, deployment constraints, and upgrade patterns.
+- Recommend changes Kubernetes SIGS can make that will provide broad benefits in a sustainable/affordable way. This would likely take the form of one or more KEPs.
+- Recommend ways users/vendors who want to maximize Kubernetes support can consume Kubernetes
+
+Any changes identified will be owned by corresponding SIGs.
+
+## Roles and Organization Management
+
+This working group adheres to the Roles and Organization Management outlined in
+[sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Timelines and Disbanding
+
+The exact timeline for this working group is hard to define at this time. If we are unable to define a common long term support definition for Kubernetes, we will disband the working group. If we are unable to define improvements related to an agreed upon definition, we will also disband the working group. In order to evaluate our progress toward the working group goals, we will provide periodic updates to the stakeholder SIGs and committees at least every six months, in addition to a working group annual report.   
```

Comment:
Would the lack of people and higher costs for providing long term support be a reason for disbanding the working group?

Reply:
I think that is more a potential outcome of the WG. If the WG determines we currently do not have the resources, nor commitments for future resources to potentially support an LTS then theres no reason to continue.

Reply:
I added this as reason for disbanding for clarity.